
load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/code-mismatch-exception
  take form code-mismatch-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/confirm-forgot-password-response
  take form confirm-forgot-password-response

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/expired-code-exception
  take form expired-code-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/internal-error-exception
  take form internal-error-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/invalid-lambda-response-exception
  take form invalid-lambda-response-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/invalid-parameter-exception
  take form invalid-parameter-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/invalid-password-exception
  take form invalid-password-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/limit-exceeded-exception
  take form limit-exceeded-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/not-authorized-exception
  take form not-authorized-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/resource-not-found-exception
  take form resource-not-found-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/too-many-failed-attempts-exception
  take form too-many-failed-attempts-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/too-many-requests-exception
  take form too-many-requests-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/unexpected-lambda-exception
  take form unexpected-lambda-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/user-lambda-validation-exception
  take form user-lambda-validation-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/user-not-confirmed-exception
  take form user-not-confirmed-exception

load @treesurf/snow/amazonaws.com/cognito-idp/2016-04-18/base/user-not-found-exception
  take form user-not-found-exception

load @treesurf/snow/base/native-string
  take form native-string

boot confirm-forgot-password, name <ConfirmForgotPassword>
  deed post
  note <Allows a user to enter a confirmation code to reset a forgotten password.>

  take x-amz-target
    like native-string
  take confirm-forgot-password-request, like confirm-forgot-password-request

  line </#X-Amz-Target=AWSCognitoIdentityProviderService.ConfirmForgotPassword>

  hint <X-Amz-Target>, loan x-amz-target

  seed json, loan confirm-forgot-password-request

  loot 200
    seed json, like confirm-forgot-password-response
    note <Success>
  loot 480
    seed json, like resource-not-found-exception
    note <ResourceNotFoundException>
  loot 481
    seed json, like unexpected-lambda-exception
    note <UnexpectedLambdaException>
  loot 482
    seed json, like user-lambda-validation-exception
    note <UserLambdaValidationException>
  loot 483
    seed json, like invalid-parameter-exception
    note <InvalidParameterException>
  loot 484
    seed json, like invalid-password-exception
    note <InvalidPasswordException>
  loot 485
    seed json, like not-authorized-exception
    note <NotAuthorizedException>
  loot 486
    seed json, like code-mismatch-exception
    note <CodeMismatchException>
  loot 487
    seed json, like expired-code-exception
    note <ExpiredCodeException>
  loot 488
    seed json, like too-many-failed-attempts-exception
    note <TooManyFailedAttemptsException>
  loot 489
    seed json, like invalid-lambda-response-exception
    note <InvalidLambdaResponseException>
  loot 490
    seed json, like too-many-requests-exception
    note <TooManyRequestsException>
  loot 491
    seed json, like limit-exceeded-exception
    note <LimitExceededException>
  loot 492
    seed json, like user-not-found-exception
    note <UserNotFoundException>
  loot 493
    seed json, like user-not-confirmed-exception
    note <UserNotConfirmedException>
  loot 494
    seed json, like internal-error-exception
    note <InternalErrorException>